packages/ui-components/src/components/text-field/text-field.scss

```diff
@@ -0,0 +1,98 @@
+@import '../../assets/styles/globals';
+
+$defaultInputHeight: 35px;
+
+.label,
+.error {
+	@include kv-font-label-small-uppercase-semibold
+}
+
+.label {
+	color: kv-color('text');
+}
+
+.error {
+	color: kv-color(error);
+}
+
+.flex-align-center {
+	display: flex;
+	align-content: center;
+}
+
+.text-container {
+	margin-left: $spacing-2x;
+
+	.text-elem {
+		width: 100%;
+		height: 16px;
+		margin-top: $spacing;
+
+		.flex-align-center {
+			display: flex;
+			align-content: center;
+		}
+
+		color: kv-color(primary, base);
+		@include kv-font-label-small-regular;
+
+		svg-icon {
+			margin-right: $spacing;
+		}
+	}
+
+	.has-error {
+		color: kv-color(error);
+	}
+}
+
+input {
+	@include kv-font-span-regular;
+
+	color: kv-color(primary);
+	width: 100%;
+	padding: $spacing-2x;
+	outline: 0;
+	border-radius: 2px;
+	border: solid 1px kv-color(primary);
+
+	&.has-error {
+		border-color: kv-color(error);
+	}
+
+	&[disabled] {
+		background-color: kv-color(primary, light);
+		color: kv-color(primary, dark);
+	}
+
+	&::placeholder {
+		color: kv-color(primary, light);
+	}
+
+	&:focus {
+		border-color: kv-color(primary, dark);
+		color: kv-color(primary, dark);
+	}
+}
+
+.input-container {
+	position: relative;
+	margin-top: $spacing;
+
+	::ng-deep svg.svg-icon {
+		position: absolute;
+		top: 0;
+		right: $spacing-2x;
+		height: 100%;
+		cursor: pointer;
+		user-select: none;
+	}
+}
+
+.input-container-loading {
+	display: flex;
+	height: $defaultInputHeight;
+	margin-top: $spacing;
+
+	@include ghost-animation;
```

Comment:
I think the ghost animation isn't working properly, please check

Reply:
check the mixin's name